glean_parser/lint.py

```diff
@@ -262,6 +262,28 @@ def check_expired_metric(
         yield ("Metric has expired. Please consider removing it.")
 
 
+def check_redundant_ping(
+    pings: pings.Ping, parser_config: Dict[str, Any]
+) -> LintGenerator:
+    """
+    Check if the pings contains 'ping' as the prefix or suffix, or 'ping' or 'custom'
+    """
+    ping_words = _split_words(pings.name)
```

Comment:
Unfortunately pings can contain dashes (`-`) and `_split_words` doesn't split on that.
Luckily it should be easy to fix: add the `-` to the function like this:

```
def _split_words(name: str) -> List[str]:
    """
    Helper function to split words on either `.` or `_`.
    """
    return re.split("[._-]", name)
```

This should not break anything further.

Reply:
I see, sure I will include '-' to the built-in function split